tutorials/basic-topics/sorting/selection-sort.md

```diff
@@ -11,4 +11,110 @@ keywords:
   - selection sort
 ---
 
-<TutorialAuthors names="@TBC"/>
+<TutorialAuthors names="@Bobliuuu"/>
+
+## Overview
+
+Selection sort is a commonly used comparison-based sorting algorithm, that is also very simple to implement. 
+Selection sort works on the premise that two subarrays are maintained: one which is **sorted**, and one which is **unsorted**. 
+Each step, one more element of the array gets sorted, until the entire array is sorted. 
+
+- **subarray**: A subsequence or slice of an array
+
+## Algorithm
+
+The concept of selection sort is that each time, we find the minimum element in the unsorted subarray, and we put it at the end of the sorted subarray. 
```

Comment:
Only end of the sorted subarray ? 
Selection sort can't work by placing smaller numbers and moving forward ?

Reply:
Yes, but smaller numbers must be placed at the end of the sorted subarray. The sorted subarray maintains the smallest values, so the next smallest value in the unsorted subarray is definitely the largest value in the sorted one. 

An example: 

![image](https://user-images.githubusercontent.com/54565411/194091239-364d0a85-4ad1-4a05-a742-ec351fd256df.png)